airlock/templates/file_browser/group.html

```diff
@@ -1,22 +1,21 @@
 {% #card title=path_item.name container=True %}
-  {% if show_c3 %}
```

Comment:
The diff looks like lots has changed - actually it's just this line that's moved to line 18 so it wraps just the comments form.